CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Why's there a newline at the top of the file?

Reply:
It's just how I write text files, I removed it now